modules/core/src/test/scala/xenomorph/Samples.scala

```diff
@@ -1,31 +1,28 @@
 package xenomorph
 
-import scalaz.syntax.apply._
-
-import monocle.Iso
 import monocle.macros._
-
-import org.joda.time.Instant
-
+import shapeless.HNil
 import xenomorph.Schema._
 import xenomorph.json.JType._
 
-import shapeless.HNil
+import scalaz.syntax.apply._
 
 package samples {
+
+
   @Lenses case class Person(
     name: String,
-    birthDate: Instant,
+    birthDate: Long,
```

Comment:
this is the thing in this PR I'm the least happy with, but I figured I'll fix it later

Reply:
Solved in 4c1c895. I had to change the generator to take into account a magic value and avoid throwing. I think it'll be a good example for how to compose different primitive sets, and avoid to pollute the "vanilla schema"